<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Filtrer</class>
 <widget class="QDialog" name="Filtrer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>527</width>
    <height>360</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Filtrer</string>
  </property>
  <widget class="QPushButton" name="Rechercher">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>90</y>
     <width>80</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Rechercher</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Entrer La valeur</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtRechercher">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>90</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>451</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Rechercher par </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtVariable">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>60</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Rechercher par : nom,prenom,CNE,massar,dateN</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
